tests/runtime/dwarf

```diff
@@ -45,6 +45,35 @@ REQUIRES_FEATURE dwarf
 TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
 
+NAME uprobe arg by index - 128-bits integer
+PROG uprobe:./testprogs/uprobe_test:uprobeFunctionUint128 { printf("x = %x\ny = %x\nz = %x\nw = %x\n", arg0, arg1, arg2, arg3); exit(); }
+EXPECT x = 9abcdef0
+       y = efefefef
+       z = cdcdcdcd
+       w = abababab
+REQUIRES_FEATURE dwarf
+REQUIRES bash -c "exit 1" # SKIP: 128-bits arguments are not supported, see #3236
```

Comment:
Nit: I'm not sure if I like the use of REQUIRES for tests that we don't support at all (yet). Maybe it's better to just comment out the whole thing and add a TODO with the issue. It's more noticeable when looking at the code IMHO.